### PR TITLE
test_vector2_length.py

### PR DESCRIPTION
**Pull request by Maowse**
Added additional tests for the length function

### CODE REVIEW

**Comment by pathunstrom** [comment]
This is a great set of tests!

Could I get you to take out the unittest and use test functions? We'll be using pytest going forward.

Bonus would be to parameterize them, but that work can be done going forward.

**Comment by Maowse** [comment]
I removed unittest and used parametrized testing from pytest, please let me know if anything needs changed (or if I did something incorrectly!)

File: tests/test_vector2_length.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import ppb_vector
+import pytest
+
```
Comment (pathunstrom) [inline tests/test_vector2_length.py]: Need one more line of white space here.

File: tests/test_vector2_length.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+@pytest.mark.parametrize("x, y, expected", [
+    (6, 8, 10),
+    (8, 6, 10),
+    (0, 0, 0),
+    (-6, -8, 10),
+    (1, 2, 2.23606797749979)
+])
+
```
Comment (pathunstrom) [inline tests/test_vector2_length.py]: And this white space should be removed a decorator should end on the line immediately preceding the function definition.

**Comment by pathunstrom** [comment]
I would normally do:

    import pytest

    import ppb_vector

Because of a basic rule of grouping imports into three things: standard library imports, third party imports, local imports. In this case, ppb_vector counts as "local" because that's what we're testing.

I don't think this is a hard requirement and the way you did it is actually fine!

**Pull request merged**